pyproject.toml

```diff
@@ -1,7 +1,5 @@
 [build-system]
-requires = [
-    "setuptools>=38.2.5",
-    "setuptools_scm",
-    "wheel",
-]
+requires = ["setuptools>=42", "setuptools_scm[toml]>=3.4", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
```

Comment:
Does this empty section have an effect of some kind?

Reply:
It does.  It's the equivalent of putting `use_scm_version=True` in `setup.py`.

https://pypi.org/project/setuptools-scm/